helevents/models.py

```diff
@@ -130,10 +130,16 @@ def get_editable_events(self, queryset):
             else:
                 return queryset.none()
 
+        if registration_admins_can_edit:
```

Comment:
This check could in its own method; `get_editable_events_for_registration` 

And adding a use of it in the registration [code below](https://github.com/City-of-Helsinki/linkedevents/pull/768/files#diff-d5b4af3577122a1d372d2e67a5845e433b9ec237a5b234b1097aeb5b8509ec64R121)

Reply:
Added separate `get_editable_events_for_registration` method  in commit https://github.com/City-of-Helsinki/linkedevents/pull/768/commits/81d24f2874a916092c848e1236e86b597ec8e64c